src/test/java/seedu/address/commons/core/ConfigTest.java

```diff
@@ -37,6 +39,33 @@ public void toString_defaultObject_stringReturned() {
     public void equalsMethod() {
         assertNotNull(defaultConfig);
         assertTrue(defaultConfig.equals(defaultConfig));
+        assertFalse(defaultConfig.equals(null));
```

Comment:
small nitpick but can add the comments + the different class -> returns false thing